src/js/taskList.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable */
+/* Main Class to set local storage */
+
+// import Task from "./task";
+
+export default class {
+  setList(tasks) {
+    return localStorage.setItem('tasks', JSON.stringify(tasks));
+  }
+
+  getList() {
+    return localStorage.getItem('tasks') ? JSON.parse(localStorage.getItem('tasks')) : [];
+  }
+
+  reOrderTaskIndex(tasks) {
+    // using destructuring and map
+    // const reorderedTasks = tasks.map(({ completed, description }, i) => new Task(i + 1, completed, description) );
+    // return this.setList(reorderedTasks);
```

Comment:
please do not leave unused codes as a comment in your codebase. unused codes as comments make your code not clean.

Reply:
Thanks for the feedback. 🥇